.github/CHANGELOG.md

```diff
@@ -8,6 +8,35 @@
 
 <h3>Improvements</h3>
 
+<h4>Broadcasting over multi-dimensional inputs to KerasLayer and TorchLayer</h4>
+
+* Most layers in Pytorch or Keras accept arbitrary dimension inputs, where each dimension barring the last (in the case where the actual weight function of the layer operates     on one-dimensional vectors) is broadcast over. This is now also supported by KerasLayer and TorchLayer [(#1062)](https://github.com/PennyLaneAI/pennylane/pull/1062).
+
+  Example use:
+  
+  ```python
+  dev = qml.device("default.qubit", wires=4)
+
+  x = tf.ones((5, 4, 4))
+
+  @qml.qnode(dev)
+  def layer(weights, inputs):
+
+      qml.templates.AngleEmbedding(inputs, wires=range(4))
+      qml.templates.StronglyEntanglingLayers(weights, wires=range(4))
+      return [qml.expval(qml.PauliZ(i)) for i in range(4)]
+
+  qlayer = qml.qnn.KerasLayer(layer, {"weights": (4, 4, 3)}, output_dim=4)
+
+  out = qlayer(x)
+  
+  print(out.shape)
+  ```
+
+  The output tensor has the following shape:
+
+  >>> out.shape
+  (5, 4, 4)
```

Comment:
This will need to become, otherwise `>>>` is interpreted as three nested block quotes.
<pre>```pycon
>>> out.shape
(5, 4, 4)
```</pre>